_includes/menu.html

```diff
@@ -50,4 +50,4 @@
       {% endif %}
     </li>
   {% endfor %}
-</ul>
+</ul>
```

Comment:
yeah thanks for the tip! The difference in our setups is something of a hassle. I personally keep a newline while others remove them - leading to these unclean diffs. pinging @dhaavi since he's the one who can define guidelines for this ;)